### PR TITLE
Add contribution docs

### PR DESCRIPTION
**Pull request by bpleslie**
Adds `CONTRIBUTING.md` per discussion in https://github.com/faker-ruby/faker-bot/issues/40

### CODE REVIEW

**Review by vbrazo — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+We love pull requests. Here's a quick guide:
+
+1. Fork the repo.
+
+2. Run the tests. We only take pull requests with passing tests, and it's great to know that you have a clean slate: `bundle && bundle exec rake`
+
+3. Please add a test for your change. Only refactoring and documentation changes require no new tests. If you are adding functionality or fixing a bug, we need a test! We use [Minitest](https://github.com/seattlerb/minitest) in this project.
```
Comment (akabiru) [inline CONTRIBUTING.md]:
```suggestion
3. Please add a test for your change. Only refactoring and documentation changes require no new tests. If you are adding functionality or fixing a bug, we need a test! We use [RSpec](https://github.com/rspec/rspec) in this project.
```

File: README.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/faker-ruby/faker-bot. This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
+See [CONTRIBUTING.md](https://github.com/faker-ruby/faker-bot/blob/master/CONTRIBUTING.md).
```
Comment (akabiru) [inline README.md]: I think we should retain the parts on keeping to the contributor covenant.

```suggestion
Bug reports and pull requests are welcome on GitHub at https://github.com/faker-ruby/faker-bot. See [CONTRIBUTING.md](https://github.com/faker-ruby/faker-bot/blob/master/CONTRIBUTING.md).
This project is intended to be a safe, welcoming space for collaboration, and contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct.
```

**Review by akabiru — COMMENTED** [review]
Thanks for this Brad! 🙌 Just some few remarks from me. 🙂

**Comment by bpleslie** [comment]
@akabiru I applied your suggested changes. Let me know if you see anything else

**Review by akabiru — APPROVED** [review]
LGTM! 👍 Thanks @bpleslie ! 🙌

**Pull request merged**